crm_claim_rma/models/crm_claim.py

```diff
@@ -0,0 +1,206 @@
+        claim_type = self.claim_type
+        claim_date = self.date
+        warehouse = self.warehouse_id
+        company = self.company_id
+        create_lines = context.get('create_lines')
+
+        def warranty_values(invoice, product):
+            values = {}
+            try:
+                warranty = claim_line._warranty_limit_values(
+                    invoice, claim_type, product, claim_date)
+            except (InvoiceNoDate, ProductNoSupplier):
+                # we don't mind at this point if the warranty can't be
+                # computed and we don't want to block the user
+                values.update({'guarantee_limit': False, 'warning': False})
+            else:
+                values.update(warranty)
+
+            warranty_address = claim_line._warranty_return_address_values(
+                product, company, warehouse)
+            values.update(warranty_address)
+            return values
+
+        if create_lines:  # happens when the invoice is changed
+            claim_lines = []
+            for invoice_line in self.invoice_id.invoice_line_ids:
+                location_dest = claim_line.get_destination_location(
+                    invoice_line.product_id, warehouse)
+                line = {
+                    'name': invoice_line.name,
+                    'claim_origin': "none",
+                    'invoice_line_id': invoice_line.id,
+                    'product_id': invoice_line.product_id.id,
+                    'product_returned_quantity': invoice_line.quantity,
+                    'unit_sale_price': invoice_line.price_unit,
+                    'location_dest_id': location_dest.id,
+                    'state': 'draft',
+                }
+                line.update(warranty_values(invoice_line.invoice_id,
+                                            invoice_line.product_id))
+                claim_lines.append((0, 0, line))
+
+            value = self._convert_to_cache(
+                {'claim_line_ids': claim_lines}, validate=False)
+            self.update(value)
+
+        if self.invoice_id:
+            self.delivery_address_id = self.invoice_id.partner_id.id
+
+    @api.model
+    def message_get_reply_to(self, res_ids, default=None):
+        """ Override to get the reply_to of the parent project.
+        """
+        results = dict.fromkeys(res_ids, default or False)
+        if res_ids:
+            claims = self.browse(res_ids)
+            results.update({
+                claim.id: self.env['crm.team'].message_get_reply_to(
+                    [claim.team_id], default
+                )[claim.team_id] for claim in claims if claim.team_id
+            })
+
+        return results
+
+    @api.multi
+    def message_get_suggested_recipients(self):
+        recipients = super(CrmClaim, self).message_get_suggested_recipients()
+        try:
+            for claim in self:
+                if claim.partner_id:
+                    self._message_add_suggested_recipient(
```

Comment:
Should be `claim._message_add_suggested_recipient(recipients, partner=claim.partner_id, reason=_('...')` for this method and the one below.